src/alire/alire-externals-lists.adb

```diff
@@ -13,7 +13,7 @@ package body Alire.Externals.Lists is
                     return Containers.Release_Set
    is
    begin
-      Trace.Info ("Looking for external crate: " & (+Name));
+      Trace.Detail ("Looking for external crate: " & (+Name));
```

Comment:
I actually changed that on purpose when working on `msys2` because the operation for external crate was taking a long time and it felt as a user that `alr` was stuck. I am talking several minutes here to download and install dependencies in `msys2`.

As a user I prefer to have some information that something is going on.

Reply:
Ah, I see. I wondered how I had forgotten about that making into the tests. All right then.